2022-08-13 02:06:55,520 [main] INFO  [c.b.miniweibo.monitor.MiniweiboMonitorApplication] StartupInfoLogger.java:55 - Starting MiniweiboMonitorApplication using Java 1.8.0_91 on DESKTOP-KGK70BV with PID 12672 (D:\MyProjects\miniweibo\miniweibo\miniweibo\miniweibo-visual\miniweibo-monitor\target\classes started by Administrator in D:\MyProjects\miniweibo\miniweibo)
2022-08-13 02:06:55,523 [main] INFO  [c.b.miniweibo.monitor.MiniweiboMonitorApplication] SpringApplication.java:634 - No active profile set, falling back to 1 default profile: "default"
2022-08-13 02:06:56,357 [main] INFO  [o.springframework.cloud.context.scope.GenericScope] GenericScope.java:283 - BeanFactory id=ec849420-672f-3261-a17e-9021107a6535
2022-08-13 02:06:56,372 [main] INFO  [c.u.j.c.EnableEncryptablePropertiesBeanFactoryPostProcessor] EnableEncryptablePropertiesBeanFactoryPostProcessor.java:50 - Post-processing PropertySource instances
2022-08-13 02:06:56,436 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource configurationProperties [org.springframework.boot.context.properties.source.ConfigurationPropertySourcesPropertySource] to AOP Proxy
2022-08-13 02:06:56,437 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource servletConfigInitParams [org.springframework.core.env.PropertySource$StubPropertySource] to EncryptablePropertySourceWrapper
2022-08-13 02:06:56,438 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource servletContextInitParams [org.springframework.core.env.PropertySource$StubPropertySource] to EncryptablePropertySourceWrapper
2022-08-13 02:06:56,438 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource systemProperties [org.springframework.core.env.PropertiesPropertySource] to EncryptableMapPropertySourceWrapper
2022-08-13 02:06:56,438 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource systemEnvironment [org.springframework.boot.env.SystemEnvironmentPropertySourceEnvironmentPostProcessor$OriginAwareSystemEnvironmentPropertySource] to EncryptableMapPropertySourceWrapper
2022-08-13 02:06:56,438 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource random [org.springframework.boot.env.RandomValuePropertySource] to EncryptablePropertySourceWrapper
2022-08-13 02:06:56,439 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource cachedrandom [org.springframework.cloud.util.random.CachedRandomPropertySource] to EncryptablePropertySourceWrapper
2022-08-13 02:06:56,439 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource springCloudClientHostInfo [org.springframework.core.env.MapPropertySource] to EncryptableMapPropertySourceWrapper
2022-08-13 02:06:56,439 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource miniweibo-monitor-dev.yml [org.springframework.boot.env.OriginTrackedMapPropertySource] to EncryptableMapPropertySourceWrapper
2022-08-13 02:06:56,439 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource application-dev.yml [org.springframework.boot.env.OriginTrackedMapPropertySource] to EncryptableMapPropertySourceWrapper
2022-08-13 02:06:56,439 [main] INFO  [c.u.j.EncryptablePropertySourceConverter] EncryptablePropertySourceConverter.java:38 - Converting PropertySource Config resource 'class path resource [application.yml]' via location 'optional:classpath:/' [org.springframework.boot.env.OriginTrackedMapPropertySource] to EncryptableMapPropertySourceWrapper
2022-08-13 02:06:56,506 [main] INFO  [c.u.j.filter.DefaultLazyPropertyFilter] DefaultLazyPropertyFilter.java:31 - Property Filter custom Bean not found with name 'encryptablePropertyFilter'. Initializing Default Property Filter
2022-08-13 02:06:56,651 [main] WARN  [io.undertow.websockets.jsr] Bootstrap.java:68 - UT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used
2022-08-13 02:06:56,667 [main] INFO  [io.undertow.servlet] ServletContextImpl.java:382 - Initializing Spring embedded WebApplicationContext
2022-08-13 02:06:56,667 [main] INFO  [o.s.b.w.s.c.ServletWebServerApplicationContext] ServletWebServerApplicationContext.java:292 - Root WebApplicationContext: initialization completed in 1110 ms
2022-08-13 02:06:56,741 [main] INFO  [c.u.j.resolver.DefaultLazyPropertyResolver] DefaultLazyPropertyResolver.java:31 - Property Resolver custom Bean not found with name 'encryptablePropertyResolver'. Initializing Default Property Resolver
2022-08-13 02:06:56,743 [main] INFO  [c.u.j.detector.DefaultLazyPropertyDetector] DefaultLazyPropertyDetector.java:30 - Property Detector custom Bean not found with name 'encryptablePropertyDetector'. Initializing Default Property Detector
2022-08-13 02:06:56,744 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:32 - String Encryptor custom Bean not found with name 'jasyptStringEncryptor'. Initializing Default String Encryptor
2022-08-13 02:06:56,749 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:58 - Encryptor config not found for property jasypt.encryptor.algorithm, using default value: PBEWithMD5AndDES
2022-08-13 02:06:56,749 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:58 - Encryptor config not found for property jasypt.encryptor.keyObtentionIterations, using default value: 1000
2022-08-13 02:06:56,750 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:58 - Encryptor config not found for property jasypt.encryptor.poolSize, using default value: 1
2022-08-13 02:06:56,750 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:58 - Encryptor config not found for property jasypt.encryptor.providerName, using default value: null
2022-08-13 02:06:56,750 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:58 - Encryptor config not found for property jasypt.encryptor.providerClassName, using default value: null
2022-08-13 02:06:56,750 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:58 - Encryptor config not found for property jasypt.encryptor.saltGeneratorClassname, using default value: org.jasypt.salt.RandomSaltGenerator
2022-08-13 02:06:56,751 [main] INFO  [c.u.j.encryptor.DefaultLazyEncryptor] DefaultLazyEncryptor.java:58 - Encryptor config not found for property jasypt.encryptor.stringOutputType, using default value: base64
2022-08-13 02:06:57,082 [main] INFO  [o.s.security.web.DefaultSecurityFilterChain] DefaultSecurityFilterChain.java:55 - Will secure any request with [org.springframework.security.web.session.DisableEncodeUrlFilter@70f4abc1, org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@26d028f7, org.springframework.security.web.context.SecurityContextPersistenceFilter@300a38e7, org.springframework.security.web.header.HeaderWriterFilter@6d420cdd, org.springframework.security.web.authentication.logout.LogoutFilter@41582899, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@402d6012, org.springframework.security.web.authentication.www.BasicAuthenticationFilter@79d4ff58, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@2bb0e277, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@6f24ce45, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@a2df0d5, org.springframework.security.web.session.SessionManagementFilter@78128dd0, org.springframework.security.web.access.ExceptionTranslationFilter@44abdb1f, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@6dcf7b6a]
2022-08-13 02:06:57,286 [main] WARN  [o.s.b.a.t.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration] ThymeleafAutoConfiguration.java:103 - Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2022-08-13 02:06:58,237 [main] INFO  [o.s.b.actuate.endpoint.web.EndpointLinksResolver] EndpointLinksResolver.java:58 - Exposing 18 endpoint(s) beneath base path '/actuator'
2022-08-13 02:06:58,366 [main] INFO  [io.undertow] Undertow.java:120 - starting server: Undertow - 2.2.18.Final
2022-08-13 02:06:58,371 [main] INFO  [org.xnio] Xnio.java:95 - XNIO version 3.8.7.Final
2022-08-13 02:06:58,377 [main] INFO  [org.xnio.nio] NioXnio.java:58 - XNIO NIO Implementation Version 3.8.7.Final
2022-08-13 02:06:58,399 [main] INFO  [org.jboss.threads] Version.java:52 - JBoss Threads version 3.1.0.Final
2022-08-13 02:06:58,428 [main] INFO  [o.s.boot.web.embedded.undertow.UndertowWebServer] UndertowWebServer.java:119 - Undertow started on port(s) 5001 (http)
2022-08-13 02:06:58,549 [main] INFO  [c.a.cloud.nacos.registry.NacosServiceRegistry] NacosServiceRegistry.java:75 - nacos registry, DEFAULT_GROUP miniweibo-monitor 192.168.10.104:5001 register finished
2022-08-13 02:06:58,554 [XNIO-1 task-1] INFO  [io.undertow.servlet] ServletContextImpl.java:382 - Initializing Spring DispatcherServlet 'dispatcherServlet'
2022-08-13 02:06:58,554 [XNIO-1 task-1] INFO  [org.springframework.web.servlet.DispatcherServlet] FrameworkServlet.java:525 - Initializing Servlet 'dispatcherServlet'
2022-08-13 02:06:58,556 [XNIO-1 task-1] INFO  [org.springframework.web.servlet.DispatcherServlet] FrameworkServlet.java:547 - Completed initialization in 1 ms
2022-08-13 02:06:58,612 [Nacos-Watch-Task-Scheduler-1] INFO  [d.c.b.a.s.c.discovery.InstanceDiscoveryListener] InstanceDiscoveryListener.java:138 - Instance '272ed2ac5d55' missing in DiscoveryClient services and will be removed.
2022-08-13 02:06:58,769 [main] INFO  [c.b.miniweibo.monitor.MiniweiboMonitorApplication] StartupInfoLogger.java:61 - Started MiniweiboMonitorApplication in 4.032 seconds (JVM running for 4.746)
2022-08-13 02:06:58,777 [main] INFO  [c.a.cloud.nacos.refresh.NacosContextRefresher] NacosContextRefresher.java:105 - listening config: dataId=miniweibo-monitor-dev.yml, group=DEFAULT_GROUP
2022-08-13 02:06:58,779 [main] INFO  [c.a.cloud.nacos.refresh.NacosContextRefresher] NacosContextRefresher.java:105 - listening config: dataId=application-dev.yml, group=DEFAULT_GROUP
2022-08-13 02:17:40,081 [reactor-http-nio-9] INFO  [d.c.boot.admin.server.services.StatusUpdater] StatusUpdater.java:130 - Couldn't retrieve status for Instance(id=08d0a5682c4b, version=3, registration=Registration(name=miniweibo-upms-biz, managementUrl=http://192.168.10.104:4000/actuator, healthUrl=http://192.168.10.104:4000/actuator/health, serviceUrl=http://192.168.10.104:4000, source=discovery), registered=true, statusInfo=StatusInfo(status=DOWN, details={sentinel={status=UP, details={dataSource={}, enabled=true, dashboard={status=UP}}}, nacosConfig={status=UP}, diskSpace={status=UP, details={total=456998776832, free=110482210816, threshold=10485760, exists=true}}, ping={status=UP}, discoveryComposite={status=UP, components={discoveryClient={status=UP, details={services=[miniweibo-upms-biz, miniweibo-auth, miniweibo-monitor, miniweibo-xxl-job-admin, miniweibo-gateway, miniweibo-sentinel-dashboard]}}}}, refreshScope={status=UP}, nacosDiscovery={status=UP}, db={status=DOWN, details={error=org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'220.112.20.63' (using password: YES)}}, redis={status=UP, details={version=4.0.14}}}), statusTimestamp=2022-08-12T18:06:59.915Z, info=Info(values={git={commit={time=2022-06-07T01:46:17Z, id=61f41b9}, branch=61f41b936c46a601ed75197d281f918af4d105d4}}), endpoints=Endpoints(endpoints={sentinel=Endpoint(id=sentinel, url=http://192.168.10.104:4000/actuator/sentinel), caches=Endpoint(id=caches, url=http://192.168.10.104:4000/actuator/caches), loggers=Endpoint(id=loggers, url=http://192.168.10.104:4000/actuator/loggers), logfile=Endpoint(id=logfile, url=http://192.168.10.104:4000/actuator/logfile), nacosconfig=Endpoint(id=nacosconfig, url=http://192.168.10.104:4000/actuator/nacosconfig), refresh=Endpoint(id=refresh, url=http://192.168.10.104:4000/actuator/refresh), health=Endpoint(id=health, url=http://192.168.10.104:4000/actuator/health), env=Endpoint(id=env, url=http://192.168.10.104:4000/actuator/env), serviceregistry=Endpoint(id=serviceregistry, url=http://192.168.10.104:4000/actuator/serviceregistry), nacosdiscovery=Endpoint(id=nacosdiscovery, url=http://192.168.10.104:4000/actuator/nacosdiscovery), heapdump=Endpoint(id=heapdump, url=http://192.168.10.104:4000/actuator/heapdump), features=Endpoint(id=features, url=http://192.168.10.104:4000/actuator/features), mappings=Endpoint(id=mappings, url=http://192.168.10.104:4000/actuator/mappings), scheduledtasks=Endpoint(id=scheduledtasks, url=http://192.168.10.104:4000/actuator/scheduledtasks), configprops=Endpoint(id=configprops, url=http://192.168.10.104:4000/actuator/configprops), beans=Endpoint(id=beans, url=http://192.168.10.104:4000/actuator/beans), threaddump=Endpoint(id=threaddump, url=http://192.168.10.104:4000/actuator/threaddump), metrics=Endpoint(id=metrics, url=http://192.168.10.104:4000/actuator/metrics), conditions=Endpoint(id=conditions, url=http://192.168.10.104:4000/actuator/conditions), info=Endpoint(id=info, url=http://192.168.10.104:4000/actuator/info)}), buildVersion=null, tags=Tags(values={}))
org.springframework.web.reactive.function.client.WebClientRequestException: Connection refused: no further information: /192.168.10.104:4000; nested exception is io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.10.104:4000
	at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	*__checkpoint ⇢ Request to GET health [DefaultWebClient]
Original Stack Trace:
		at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
		at reactor.core.publisher.MonoErrorSupplied.subscribe(MonoErrorSupplied.java:55)
		at reactor.core.publisher.Mono.subscribe(Mono.java:4397)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.MonoNext$NextSubscriber.onError(MonoNext.java:93)
		at reactor.core.publisher.MonoFlatMapMany$FlatMapManyMain.onError(MonoFlatMapMany.java:204)
		at reactor.core.publisher.SerializedSubscriber.onError(SerializedSubscriber.java:124)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.whenError(FluxRetryWhen.java:225)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenOtherSubscriber.onError(FluxRetryWhen.java:274)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.drain(FluxConcatMap.java:415)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.onNext(FluxConcatMap.java:251)
		at reactor.core.publisher.EmitterProcessor.drain(EmitterProcessor.java:537)
		at reactor.core.publisher.EmitterProcessor.tryEmitNext(EmitterProcessor.java:343)
		at reactor.core.publisher.SinkManySerialized.tryEmitNext(SinkManySerialized.java:100)
		at reactor.core.publisher.InternalManySink.emitNext(InternalManySink.java:27)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.onError(FluxRetryWhen.java:190)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.http.client.HttpClientConnect$MonoHttpConnect$ClientTransportSubscriber.onError(HttpClientConnect.java:308)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.resources.DefaultPooledConnectionProvider$DisposableAcquire.onError(DefaultPooledConnectionProvider.java:158)
		at reactor.netty.internal.shaded.reactor.pool.AbstractPool$Borrower.fail(AbstractPool.java:475)
		at reactor.netty.internal.shaded.reactor.pool.SimpleDequePool.lambda$drainLoop$9(SimpleDequePool.java:431)
		at reactor.core.publisher.FluxDoOnEach$DoOnEachSubscriber.onError(FluxDoOnEach.java:186)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.resources.DefaultPooledConnectionProvider$PooledConnectionAllocator$PooledConnectionInitializer.onError(DefaultPooledConnectionProvider.java:548)
		at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:192)
		at reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:259)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106)
		at reactor.core.publisher.Operators.error(Operators.java:198)
		at reactor.core.publisher.MonoError.subscribe(MonoError.java:53)
		at reactor.core.publisher.Mono.subscribe(Mono.java:4397)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)
		at reactor.netty.transport.TransportConnector$MonoChannelPromise.tryFailure(TransportConnector.java:534)
		at reactor.netty.transport.TransportConnector$MonoChannelPromise.setFailure(TransportConnector.java:488)
		at reactor.netty.transport.TransportConnector.lambda$doConnect$7(TransportConnector.java:223)
		at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
		at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:571)
		at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:550)
		at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
		at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
		at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:609)
		at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:117)
		at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.fulfillConnectPromise(AbstractNioChannel.java:321)
		at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:337)
		at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
		at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
		at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
		at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
		at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
		at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
		at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.10.104:4000
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:337)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:745)
2022-08-13 03:13:29,576 [parallel-1] INFO  [d.c.boot.admin.server.services.StatusUpdater] StatusUpdater.java:130 - Couldn't retrieve status for Instance(id=08d0a5682c4b, version=5, registration=Registration(name=miniweibo-upms-biz, managementUrl=http://192.168.10.104:4000/actuator, healthUrl=http://192.168.10.104:4000/actuator/health, serviceUrl=http://192.168.10.104:4000, source=discovery), registered=true, statusInfo=StatusInfo(status=DOWN, details={sentinel={status=UP, details={dataSource={}, enabled=true, dashboard={status=UP}}}, nacosConfig={status=UP}, diskSpace={status=UP, details={total=456998776832, free=110473740288, threshold=10485760, exists=true}}, ping={status=UP}, discoveryComposite={status=UP, components={discoveryClient={status=UP, details={services=[miniweibo-upms-biz, miniweibo-auth, miniweibo-monitor, miniweibo-xxl-job-admin, miniweibo-gateway, miniweibo-sentinel-dashboard]}}}}, refreshScope={status=UP}, nacosDiscovery={status=UP}, db={status=DOWN, details={error=org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'220.112.20.63' (using password: YES)}}, redis={status=UP, details={version=4.0.14}}}), statusTimestamp=2022-08-12T18:17:59.381Z, info=Info(values={git={commit={time=2022-06-07T01:46:17Z, id=61f41b9}, branch=61f41b936c46a601ed75197d281f918af4d105d4}}), endpoints=Endpoints(endpoints={sentinel=Endpoint(id=sentinel, url=http://192.168.10.104:4000/actuator/sentinel), caches=Endpoint(id=caches, url=http://192.168.10.104:4000/actuator/caches), loggers=Endpoint(id=loggers, url=http://192.168.10.104:4000/actuator/loggers), logfile=Endpoint(id=logfile, url=http://192.168.10.104:4000/actuator/logfile), nacosconfig=Endpoint(id=nacosconfig, url=http://192.168.10.104:4000/actuator/nacosconfig), refresh=Endpoint(id=refresh, url=http://192.168.10.104:4000/actuator/refresh), health=Endpoint(id=health, url=http://192.168.10.104:4000/actuator/health), env=Endpoint(id=env, url=http://192.168.10.104:4000/actuator/env), serviceregistry=Endpoint(id=serviceregistry, url=http://192.168.10.104:4000/actuator/serviceregistry), nacosdiscovery=Endpoint(id=nacosdiscovery, url=http://192.168.10.104:4000/actuator/nacosdiscovery), heapdump=Endpoint(id=heapdump, url=http://192.168.10.104:4000/actuator/heapdump), features=Endpoint(id=features, url=http://192.168.10.104:4000/actuator/features), mappings=Endpoint(id=mappings, url=http://192.168.10.104:4000/actuator/mappings), scheduledtasks=Endpoint(id=scheduledtasks, url=http://192.168.10.104:4000/actuator/scheduledtasks), configprops=Endpoint(id=configprops, url=http://192.168.10.104:4000/actuator/configprops), beans=Endpoint(id=beans, url=http://192.168.10.104:4000/actuator/beans), threaddump=Endpoint(id=threaddump, url=http://192.168.10.104:4000/actuator/threaddump), metrics=Endpoint(id=metrics, url=http://192.168.10.104:4000/actuator/metrics), conditions=Endpoint(id=conditions, url=http://192.168.10.104:4000/actuator/conditions), info=Endpoint(id=info, url=http://192.168.10.104:4000/actuator/info)}), buildVersion=null, tags=Tags(values={}))
java.util.concurrent.TimeoutException: Did not observe any item or terminal signal within 10000ms in 'map' (and no fallback has been configured)
	at reactor.core.publisher.FluxTimeout$TimeoutMainSubscriber.handleTimeout(FluxTimeout.java:295)
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	*__checkpoint ⇢ Request to GET health [DefaultWebClient]
Original Stack Trace:
		at reactor.core.publisher.FluxTimeout$TimeoutMainSubscriber.handleTimeout(FluxTimeout.java:295)
		at reactor.core.publisher.FluxTimeout$TimeoutMainSubscriber.doTimeout(FluxTimeout.java:280)
		at reactor.core.publisher.FluxTimeout$TimeoutTimeoutSubscriber.onNext(FluxTimeout.java:419)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onNext(FluxOnErrorResume.java:79)
		at reactor.core.publisher.MonoDelay$MonoDelayRunnable.propagateDelay(MonoDelay.java:271)
		at reactor.core.publisher.MonoDelay$MonoDelayRunnable.run(MonoDelay.java:286)
		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
		at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
		at java.util.concurrent.FutureTask.run(FutureTask.java)
		at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
		at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
		at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
		at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
		at java.lang.Thread.run(Thread.java:745)
2022-08-13 07:14:29,565 [parallel-3] INFO  [d.c.boot.admin.server.services.StatusUpdater] StatusUpdater.java:130 - Couldn't retrieve status for Instance(id=08d0a5682c4b, version=7, registration=Registration(name=miniweibo-upms-biz, managementUrl=http://192.168.10.104:4000/actuator, healthUrl=http://192.168.10.104:4000/actuator/health, serviceUrl=http://192.168.10.104:4000, source=discovery), registered=true, statusInfo=StatusInfo(status=DOWN, details={sentinel={status=UP, details={dataSource={}, enabled=true, dashboard={status=UP}}}, nacosConfig={status=UP}, diskSpace={status=UP, details={total=456998776832, free=110493958144, threshold=10485760, exists=true}}, ping={status=UP}, discoveryComposite={status=UP, components={discoveryClient={status=UP, details={services=[miniweibo-upms-biz, miniweibo-auth, miniweibo-monitor, miniweibo-xxl-job-admin, miniweibo-gateway, miniweibo-sentinel-dashboard]}}}}, refreshScope={status=UP}, nacosDiscovery={status=UP}, db={status=DOWN, details={error=org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'220.112.20.63' (using password: YES)}}, redis={status=UP, details={version=4.0.14}}}), statusTimestamp=2022-08-12T19:14:00.731Z, info=Info(values={git={commit={time=2022-06-07T01:46:17Z, id=61f41b9}, branch=61f41b936c46a601ed75197d281f918af4d105d4}}), endpoints=Endpoints(endpoints={sentinel=Endpoint(id=sentinel, url=http://192.168.10.104:4000/actuator/sentinel), caches=Endpoint(id=caches, url=http://192.168.10.104:4000/actuator/caches), loggers=Endpoint(id=loggers, url=http://192.168.10.104:4000/actuator/loggers), logfile=Endpoint(id=logfile, url=http://192.168.10.104:4000/actuator/logfile), nacosconfig=Endpoint(id=nacosconfig, url=http://192.168.10.104:4000/actuator/nacosconfig), refresh=Endpoint(id=refresh, url=http://192.168.10.104:4000/actuator/refresh), health=Endpoint(id=health, url=http://192.168.10.104:4000/actuator/health), env=Endpoint(id=env, url=http://192.168.10.104:4000/actuator/env), serviceregistry=Endpoint(id=serviceregistry, url=http://192.168.10.104:4000/actuator/serviceregistry), nacosdiscovery=Endpoint(id=nacosdiscovery, url=http://192.168.10.104:4000/actuator/nacosdiscovery), heapdump=Endpoint(id=heapdump, url=http://192.168.10.104:4000/actuator/heapdump), features=Endpoint(id=features, url=http://192.168.10.104:4000/actuator/features), mappings=Endpoint(id=mappings, url=http://192.168.10.104:4000/actuator/mappings), scheduledtasks=Endpoint(id=scheduledtasks, url=http://192.168.10.104:4000/actuator/scheduledtasks), configprops=Endpoint(id=configprops, url=http://192.168.10.104:4000/actuator/configprops), beans=Endpoint(id=beans, url=http://192.168.10.104:4000/actuator/beans), threaddump=Endpoint(id=threaddump, url=http://192.168.10.104:4000/actuator/threaddump), metrics=Endpoint(id=metrics, url=http://192.168.10.104:4000/actuator/metrics), conditions=Endpoint(id=conditions, url=http://192.168.10.104:4000/actuator/conditions), info=Endpoint(id=info, url=http://192.168.10.104:4000/actuator/info)}), buildVersion=null, tags=Tags(values={}))
java.util.concurrent.TimeoutException: Did not observe any item or terminal signal within 10000ms in 'map' (and no fallback has been configured)
	at reactor.core.publisher.FluxTimeout$TimeoutMainSubscriber.handleTimeout(FluxTimeout.java:295)
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	*__checkpoint ⇢ Request to GET health [DefaultWebClient]
Original Stack Trace:
		at reactor.core.publisher.FluxTimeout$TimeoutMainSubscriber.handleTimeout(FluxTimeout.java:295)
		at reactor.core.publisher.FluxTimeout$TimeoutMainSubscriber.doTimeout(FluxTimeout.java:280)
		at reactor.core.publisher.FluxTimeout$TimeoutTimeoutSubscriber.onNext(FluxTimeout.java:419)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onNext(FluxOnErrorResume.java:79)
		at reactor.core.publisher.MonoDelay$MonoDelayRunnable.propagateDelay(MonoDelay.java:271)
		at reactor.core.publisher.MonoDelay$MonoDelayRunnable.run(MonoDelay.java:286)
		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
		at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
		at java.util.concurrent.FutureTask.run(FutureTask.java)
		at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
		at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
		at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
		at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
		at java.lang.Thread.run(Thread.java:745)
2022-08-13 12:13:01,578 [reactor-http-nio-12] INFO  [d.c.boot.admin.server.services.StatusUpdater] StatusUpdater.java:130 - Couldn't retrieve status for Instance(id=08d0a5682c4b, version=9, registration=Registration(name=miniweibo-upms-biz, managementUrl=http://192.168.10.104:4000/actuator, healthUrl=http://192.168.10.104:4000/actuator/health, serviceUrl=http://192.168.10.104:4000, source=discovery), registered=true, statusInfo=StatusInfo(status=DOWN, details={sentinel={status=UP, details={dataSource={}, enabled=true, dashboard={status=UP}}}, nacosConfig={status=UP}, diskSpace={status=UP, details={total=456998776832, free=110474104832, threshold=10485760, exists=true}}, ping={status=UP}, discoveryComposite={status=UP, components={discoveryClient={status=UP, details={services=[miniweibo-upms-biz, miniweibo-auth, miniweibo-monitor, miniweibo-xxl-job-admin, miniweibo-gateway, miniweibo-sentinel-dashboard]}}}}, refreshScope={status=UP}, nacosDiscovery={status=UP}, db={status=DOWN, details={error=org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'220.112.20.63' (using password: YES)}}, redis={status=UP, details={version=4.0.14}}}), statusTimestamp=2022-08-12T23:15:00.791Z, info=Info(values={git={commit={time=2022-06-07T01:46:17Z, id=61f41b9}, branch=61f41b936c46a601ed75197d281f918af4d105d4}}), endpoints=Endpoints(endpoints={sentinel=Endpoint(id=sentinel, url=http://192.168.10.104:4000/actuator/sentinel), caches=Endpoint(id=caches, url=http://192.168.10.104:4000/actuator/caches), loggers=Endpoint(id=loggers, url=http://192.168.10.104:4000/actuator/loggers), logfile=Endpoint(id=logfile, url=http://192.168.10.104:4000/actuator/logfile), nacosconfig=Endpoint(id=nacosconfig, url=http://192.168.10.104:4000/actuator/nacosconfig), refresh=Endpoint(id=refresh, url=http://192.168.10.104:4000/actuator/refresh), health=Endpoint(id=health, url=http://192.168.10.104:4000/actuator/health), env=Endpoint(id=env, url=http://192.168.10.104:4000/actuator/env), serviceregistry=Endpoint(id=serviceregistry, url=http://192.168.10.104:4000/actuator/serviceregistry), nacosdiscovery=Endpoint(id=nacosdiscovery, url=http://192.168.10.104:4000/actuator/nacosdiscovery), heapdump=Endpoint(id=heapdump, url=http://192.168.10.104:4000/actuator/heapdump), features=Endpoint(id=features, url=http://192.168.10.104:4000/actuator/features), mappings=Endpoint(id=mappings, url=http://192.168.10.104:4000/actuator/mappings), scheduledtasks=Endpoint(id=scheduledtasks, url=http://192.168.10.104:4000/actuator/scheduledtasks), configprops=Endpoint(id=configprops, url=http://192.168.10.104:4000/actuator/configprops), beans=Endpoint(id=beans, url=http://192.168.10.104:4000/actuator/beans), threaddump=Endpoint(id=threaddump, url=http://192.168.10.104:4000/actuator/threaddump), metrics=Endpoint(id=metrics, url=http://192.168.10.104:4000/actuator/metrics), conditions=Endpoint(id=conditions, url=http://192.168.10.104:4000/actuator/conditions), info=Endpoint(id=info, url=http://192.168.10.104:4000/actuator/info)}), buildVersion=null, tags=Tags(values={}))
org.springframework.web.reactive.function.client.WebClientRequestException: Connection refused: no further information: /192.168.10.104:4000; nested exception is io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.10.104:4000
	at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	*__checkpoint ⇢ Request to GET health [DefaultWebClient]
Original Stack Trace:
		at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
		at reactor.core.publisher.MonoErrorSupplied.subscribe(MonoErrorSupplied.java:55)
		at reactor.core.publisher.Mono.subscribe(Mono.java:4397)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.MonoNext$NextSubscriber.onError(MonoNext.java:93)
		at reactor.core.publisher.MonoFlatMapMany$FlatMapManyMain.onError(MonoFlatMapMany.java:204)
		at reactor.core.publisher.SerializedSubscriber.onError(SerializedSubscriber.java:124)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.whenError(FluxRetryWhen.java:225)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenOtherSubscriber.onError(FluxRetryWhen.java:274)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.drain(FluxConcatMap.java:415)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.onNext(FluxConcatMap.java:251)
		at reactor.core.publisher.EmitterProcessor.drain(EmitterProcessor.java:537)
		at reactor.core.publisher.EmitterProcessor.tryEmitNext(EmitterProcessor.java:343)
		at reactor.core.publisher.SinkManySerialized.tryEmitNext(SinkManySerialized.java:100)
		at reactor.core.publisher.InternalManySink.emitNext(InternalManySink.java:27)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.onError(FluxRetryWhen.java:190)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.http.client.HttpClientConnect$MonoHttpConnect$ClientTransportSubscriber.onError(HttpClientConnect.java:308)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.resources.DefaultPooledConnectionProvider$DisposableAcquire.onError(DefaultPooledConnectionProvider.java:158)
		at reactor.netty.internal.shaded.reactor.pool.AbstractPool$Borrower.fail(AbstractPool.java:475)
		at reactor.netty.internal.shaded.reactor.pool.SimpleDequePool.lambda$drainLoop$9(SimpleDequePool.java:431)
		at reactor.core.publisher.FluxDoOnEach$DoOnEachSubscriber.onError(FluxDoOnEach.java:186)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.resources.DefaultPooledConnectionProvider$PooledConnectionAllocator$PooledConnectionInitializer.onError(DefaultPooledConnectionProvider.java:548)
		at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:192)
		at reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:259)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106)
		at reactor.core.publisher.Operators.error(Operators.java:198)
		at reactor.core.publisher.MonoError.subscribe(MonoError.java:53)
		at reactor.core.publisher.Mono.subscribe(Mono.java:4397)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)
		at reactor.netty.transport.TransportConnector$MonoChannelPromise.tryFailure(TransportConnector.java:534)
		at reactor.netty.transport.TransportConnector$MonoChannelPromise.setFailure(TransportConnector.java:488)
		at reactor.netty.transport.TransportConnector.lambda$doConnect$7(TransportConnector.java:223)
		at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
		at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:571)
		at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:550)
		at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
		at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
		at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:609)
		at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:117)
		at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.fulfillConnectPromise(AbstractNioChannel.java:321)
		at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:337)
		at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
		at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
		at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
		at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
		at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
		at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
		at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.10.104:4000
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:337)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:745)
2022-08-13 12:13:01,587 [reactor-http-nio-1] WARN  [d.c.boot.admin.server.services.InfoUpdater] InfoUpdater.java:91 - Couldn't retrieve info for Instance(id=08d0a5682c4b, version=9, registration=Registration(name=miniweibo-upms-biz, managementUrl=http://192.168.10.104:4000/actuator, healthUrl=http://192.168.10.104:4000/actuator/health, serviceUrl=http://192.168.10.104:4000, source=discovery), registered=true, statusInfo=StatusInfo(status=DOWN, details={sentinel={status=UP, details={dataSource={}, enabled=true, dashboard={status=UP}}}, nacosConfig={status=UP}, diskSpace={status=UP, details={total=456998776832, free=110474104832, threshold=10485760, exists=true}}, ping={status=UP}, discoveryComposite={status=UP, components={discoveryClient={status=UP, details={services=[miniweibo-upms-biz, miniweibo-auth, miniweibo-monitor, miniweibo-xxl-job-admin, miniweibo-gateway, miniweibo-sentinel-dashboard]}}}}, refreshScope={status=UP}, nacosDiscovery={status=UP}, db={status=DOWN, details={error=org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'220.112.20.63' (using password: YES)}}, redis={status=UP, details={version=4.0.14}}}), statusTimestamp=2022-08-12T23:15:00.791Z, info=Info(values={git={commit={time=2022-06-07T01:46:17Z, id=61f41b9}, branch=61f41b936c46a601ed75197d281f918af4d105d4}}), endpoints=Endpoints(endpoints={sentinel=Endpoint(id=sentinel, url=http://192.168.10.104:4000/actuator/sentinel), caches=Endpoint(id=caches, url=http://192.168.10.104:4000/actuator/caches), loggers=Endpoint(id=loggers, url=http://192.168.10.104:4000/actuator/loggers), logfile=Endpoint(id=logfile, url=http://192.168.10.104:4000/actuator/logfile), nacosconfig=Endpoint(id=nacosconfig, url=http://192.168.10.104:4000/actuator/nacosconfig), refresh=Endpoint(id=refresh, url=http://192.168.10.104:4000/actuator/refresh), health=Endpoint(id=health, url=http://192.168.10.104:4000/actuator/health), env=Endpoint(id=env, url=http://192.168.10.104:4000/actuator/env), serviceregistry=Endpoint(id=serviceregistry, url=http://192.168.10.104:4000/actuator/serviceregistry), nacosdiscovery=Endpoint(id=nacosdiscovery, url=http://192.168.10.104:4000/actuator/nacosdiscovery), heapdump=Endpoint(id=heapdump, url=http://192.168.10.104:4000/actuator/heapdump), features=Endpoint(id=features, url=http://192.168.10.104:4000/actuator/features), mappings=Endpoint(id=mappings, url=http://192.168.10.104:4000/actuator/mappings), scheduledtasks=Endpoint(id=scheduledtasks, url=http://192.168.10.104:4000/actuator/scheduledtasks), configprops=Endpoint(id=configprops, url=http://192.168.10.104:4000/actuator/configprops), beans=Endpoint(id=beans, url=http://192.168.10.104:4000/actuator/beans), threaddump=Endpoint(id=threaddump, url=http://192.168.10.104:4000/actuator/threaddump), metrics=Endpoint(id=metrics, url=http://192.168.10.104:4000/actuator/metrics), conditions=Endpoint(id=conditions, url=http://192.168.10.104:4000/actuator/conditions), info=Endpoint(id=info, url=http://192.168.10.104:4000/actuator/info)}), buildVersion=null, tags=Tags(values={}))
org.springframework.web.reactive.function.client.WebClientRequestException: Connection refused: no further information: /192.168.10.104:4000; nested exception is io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.10.104:4000
	at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	*__checkpoint ⇢ Request to GET info [DefaultWebClient]
Original Stack Trace:
		at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
		at reactor.core.publisher.MonoErrorSupplied.subscribe(MonoErrorSupplied.java:55)
		at reactor.core.publisher.Mono.subscribe(Mono.java:4397)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.MonoNext$NextSubscriber.onError(MonoNext.java:93)
		at reactor.core.publisher.MonoFlatMapMany$FlatMapManyMain.onError(MonoFlatMapMany.java:204)
		at reactor.core.publisher.SerializedSubscriber.onError(SerializedSubscriber.java:124)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.whenError(FluxRetryWhen.java:225)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenOtherSubscriber.onError(FluxRetryWhen.java:274)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.drain(FluxConcatMap.java:415)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.onNext(FluxConcatMap.java:251)
		at reactor.core.publisher.EmitterProcessor.drain(EmitterProcessor.java:537)
		at reactor.core.publisher.EmitterProcessor.tryEmitNext(EmitterProcessor.java:343)
		at reactor.core.publisher.SinkManySerialized.tryEmitNext(SinkManySerialized.java:100)
		at reactor.core.publisher.InternalManySink.emitNext(InternalManySink.java:27)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.onError(FluxRetryWhen.java:190)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.http.client.HttpClientConnect$MonoHttpConnect$ClientTransportSubscriber.onError(HttpClientConnect.java:308)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.resources.DefaultPooledConnectionProvider$DisposableAcquire.onError(DefaultPooledConnectionProvider.java:158)
		at reactor.netty.internal.shaded.reactor.pool.AbstractPool$Borrower.fail(AbstractPool.java:475)
		at reactor.netty.internal.shaded.reactor.pool.SimpleDequePool.lambda$drainLoop$9(SimpleDequePool.java:431)
		at reactor.core.publisher.FluxDoOnEach$DoOnEachSubscriber.onError(FluxDoOnEach.java:186)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:201)
		at reactor.netty.resources.DefaultPooledConnectionProvider$PooledConnectionAllocator$PooledConnectionInitializer.onError(DefaultPooledConnectionProvider.java:548)
		at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:192)
		at reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:259)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106)
		at reactor.core.publisher.Operators.error(Operators.java:198)
		at reactor.core.publisher.MonoError.subscribe(MonoError.java:53)
		at reactor.core.publisher.Mono.subscribe(Mono.java:4397)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)
		at reactor.netty.transport.TransportConnector$MonoChannelPromise.tryFailure(TransportConnector.java:534)
		at reactor.netty.transport.TransportConnector$MonoChannelPromise.setFailure(TransportConnector.java:488)
		at reactor.netty.transport.TransportConnector.lambda$doConnect$7(TransportConnector.java:223)
		at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
		at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:571)
		at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:550)
		at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
		at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
		at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:609)
		at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:117)
		at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.fulfillConnectPromise(AbstractNioChannel.java:321)
		at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:337)
		at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
		at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
		at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
		at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
		at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
		at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
		at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.10.104:4000
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:337)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:745)
2022-08-13 12:13:03,681 [Nacos-Watch-Task-Scheduler-1] INFO  [d.c.b.a.s.c.discovery.InstanceDiscoveryListener] InstanceDiscoveryListener.java:138 - Instance '08d0a5682c4b' missing in DiscoveryClient services and will be removed.
2022-08-13 12:45:03,883 [Nacos-Watch-Task-Scheduler-1] INFO  [d.c.b.a.s.c.discovery.InstanceDiscoveryListener] InstanceDiscoveryListener.java:138 - Instance '44fd2056f7cd' missing in DiscoveryClient services and will be removed.
2022-08-13 12:52:30,805 [SpringApplicationShutdownHook] INFO  [io.undertow] Undertow.java:259 - stopping server: Undertow - 2.2.18.Final
2022-08-13 12:52:30,810 [SpringApplicationShutdownHook] INFO  [io.undertow.servlet] ServletContextImpl.java:382 - Destroying Spring FrameworkServlet 'dispatcherServlet'
2022-08-13 12:52:32,838 [SpringApplicationShutdownHook] INFO  [c.a.cloud.nacos.registry.NacosServiceRegistry] NacosServiceRegistry.java:94 - De-registering from Nacos Server now...
2022-08-13 12:52:32,840 [SpringApplicationShutdownHook] INFO  [c.a.cloud.nacos.registry.NacosServiceRegistry] NacosServiceRegistry.java:114 - De-registration finished.
